feedback-suggestion/src/app/endpoints/GetFeedbackSuggestions.py

```diff
@@ -0,0 +1,52 @@
+from logging import getLogger
+from pydantic import BaseModel
+from typing import Dict, List
+from .authenticated_request import AuthRequest
+from ..extract_methods.extract_methods import extract_methods
+from ..extract_methods.method_node import MethodNode
+from ..endpoints.FeedbackSuggestionRequest import FeedbackSuggestionRequest
```

Comment:
Do the import like this:
```suggestion
from .FeedbackSuggestionRequest import get_feedbacks_for_exercise
```